### PR TITLE
Add freeradius config test

### PR DESCRIPTION
**Pull request by mazeika**
Added freeradius config test
@vinted/sre

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,8 +4,5 @@
 license          "All rights reserved"
 description      "Installs/Configures freeradius"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "1.0.14"
+version          "1.1.0"
```
Comment (vdnt) [inline metadata.rb]: patch level change

**Review by vdnt — CHANGES_REQUESTED** [review]

File: metadata.rb (hunk 1 above)
Comment (mazeika) [reply metadata.rb]: Fixed.

**Comment by mazeika** [comment]
fixed
@vinted/sre

**Review by vdnt — APPROVED** [review]

**Review by vdnt — APPROVED** [review]

File: metadata.rb [hunk 2]
```diff
@@ -4,8 +4,5 @@
 license          "All rights reserved"
 description      "Installs/Configures freeradius"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "1.0.14"
+version          "1.0.15"
 
-%w{ ubuntu centos debian }.each do |os|
```
Comment (ernestas-vinted) [inline metadata.rb]: this was in https://github.com/vinted/chef-freeradius/pull/3/files

**Comment by ernestas-vinted** [comment]
first merge this https://github.com/vinted/chef-freeradius/pull/3/files and rebase on this PR

**Comment by mazeika** [comment]
Rebased, fixed version
@vinted/sre

**Review by benner — APPROVED** [review]

**Review by ernestas-vinted — APPROVED** [review]

**Pull request merged**